test/integration/Android/FiltersTests.cs

```diff
@@ -0,0 +1,68 @@
+//Licensed under the Apache License, Version 2.0 (the "License");
+//you may not use this file except in compliance with the License.
+//See the NOTICE file distributed with this work for additional
+//information regarding copyright ownership.
+//You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+//Unless required by applicable law or agreed to in writing, software
+//distributed under the License is distributed on an "AS IS" BASIS,
+//WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//See the License for the specific language governing permissions and
+//limitations under the License.
+
+using System.Collections.Generic;
+using NUnit.Framework;
+using OpenQA.Selenium.Appium.Android;
+using OpenQA.Selenium.Appium;
+
+namespace Appium.Net.Integration.Tests.helpers
+{
+    [TestFixture]
+    public class FiltersTests
+    {
+        private AndroidDriver _driver;
+
+        [OneTimeSetUp]
+        public void BeforeAll()
+        {
+            var capabilities = Env.ServerIsRemote()
+                ? Caps.GetAndroidUIAutomatorCaps(Apps.Get("androidApiDemos"))
+                : Caps.GetAndroidUIAutomatorCaps(Apps.Get("androidApiDemos"));
+            var serverUri = Env.ServerIsRemote() ? AppiumServers.RemoteServerUri : AppiumServers.LocalServiceUri;
+            _driver = new AndroidDriver(serverUri, capabilities, Env.InitTimeoutSec);
+            _driver.Manage().Timeouts().ImplicitWait = Env.ImplicitTimeoutSec;
+        }
+
+        [SetUp]
+        public void SetUp()
+        {
+            _driver.StartActivity("io.appium.android.apis", ".ApiDemos");
```

Comment:
which server API does this method call under the hood?

Reply:
could you please provide the full URL? I am asking because `/session/:sessionId/appium/device/start_activity` is marked as deprecated and will be deleted in appium3

Reply:
I just removed that SetUp, it's not needed here.